Packs/CVE_2024_47575/Playbooks/playbook-CVE-2024-47575_-_FortiManager_Authentication_Bypass.yml

```diff
@@ -0,0 +1,542 @@
+tags:
+- FortiJump
+- CVE-2024-47575
+- FortiManager
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: e638d977-c697-44e9-8caf-9711598be275
+    type: start
+    task:
+      id: e638d977-c697-44e9-8caf-9711598be275
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 60
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 3e555db3-5358-4a2c-8541-3977d0dfa466
+    type: regular
+    task:
+      id: 3e555db3-5358-4a2c-8541-3977d0dfa466
+      version: -1
+      name: Extract Indicators
+      description: commands.local.cmd.extract.indicators
+      script: Builtin|||extractIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      text:
+        simple: ${inputs.rawIoCs}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 23a45588-c954-4d0c-8f0b-61d032408181
+    type: title
+    task:
+      id: 23a45588-c954-4d0c-8f0b-61d032408181
+      version: -1
+      name: Create and Tag Indicators
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "3"
+      - "4"
+      - "9"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 41432b0b-182a-425e-847d-0f294360c5d5
+    type: regular
+    task:
+      id: 41432b0b-182a-425e-847d-0f294360c5d5
+      version: -1
+      name: IP Indicators
+      description: commands.local.cmd.new.indicator
+      script: Builtin|||createNewIndicator
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      tags:
+        simple: FortiManager, CVE-2024-47575, FortiJump
```

Comment:
In the future the tag "FortiManager" might be confusing. maybe remove it or change to "FortiManager vulnerability"?

Reply:
same for the email indicator

Reply:
NP, Done.